src/velero/zarf.yaml

```diff
@@ -1,11 +1,42 @@
 kind: ZarfPackageConfig
 metadata:
   name: uds-core-velero
-  description: "Example description"
+  description: "UDS Core Velero"
+  url: https://velero.io/
 
 components:
-  - name: test
-    actions:
-      onCreate:
-        after:
-          - cmd: echo "What can Doug do for you?"
+  - name: velero
+    required: true
+    only:
+      flavor: upstream
+    import:
+      path: common
+    charts:
+      - name: velero
+        valuesFiles:
+          - values/upstream-values.yaml
+    images:
+      - velero/velero:v1.13.0
+      - velero/velero-plugin-for-csi:v0.7.0
+      - velero/velero-plugin-for-aws:v1.9.0
+      - velero/velero-plugin-for-microsoft-azure:v1.8.2
```

Comment:
Commenting to get other opinions here - we currently have the chart configured to expect AWS storage (s3). Someone could certainly helm override to use a different provider, so to some extent makes sense to include the other plugins. Just wary of including extra images that aren't used - these ones are likely small so makybe less of a problem?

Reply:
I (obviously) leaned towards including them because we've needed them (AFS driver and the generic CSI driver) in past efforts. I think the safe assumption is that the majority of use cases will be S3 or S3 compatible which is why the default values for the pkg make it easy to configure for that use case.. but I'm open to the argument that since we don't currently need it maybe we shouldn't include them. It's not like it would be difficult to add them back later as needed 🤷 

FWIW, these are the image sizes:
```
$ docker images
REPOSITORY                                                           TAG            IMAGE ID       CREATED         SIZE
velero/velero-plugin-for-csi                                         v0.7.0         af809012f8ca   5 weeks ago     55.6MB
velero/velero-plugin-for-aws                                         v1.9.0         1f561b2b60e1   5 weeks ago     85MB
velero/velero-plugin-for-microsoft-azure                             v1.8.2         4d08719dba26   3 months ago    54.5MB
```

Reply:
Leaning towards drop the extras (for now) and explicitly call out the s3 compatible assumption. Still would be an option if the image was sideloaded (spearate zarf package) but would make it more clear what our opinion is with the default values, and keep things slimmer to start. Very happy to re-evaluate as this rolls out to more users if we find that csi/azure are common asks.

Reply:
Sounds good, I'll drop them and add a note in the README.